src/resin.cpp

```diff
@@ -15,18 +15,26 @@ void Resin::Setup(const char* applicationUUID, const char* ssid, const char* pas
 
     WiFi.setAutoConnect(true);
     WiFi.setAutoReconnect(true);
-    WiFi.begin(_ssid, _password);
+    WiFi.begin(_ssid.c_str(), _password.c_str());
     while (WiFi.status() != WL_CONNECTED) {
         delay(100);
     }
 
     _httpServer = ESP8266WebServer(80);
     _httpUpdater = ESP8266HTTPUpdateServer();
 
-    _httpServer.on("/id", [&](){
+    _httpServer.on("/id", HTTP_GET, [&]() {
         _httpServer.send(200, "text/plain", _applicationUUID);
     });
 
+    _httpServer.on("/env", HTTP_POST, [&]() {
+        DynamicJsonBuffer jsonBuffer;
+        JsonObject& root = jsonBuffer.parseObject(_httpServer.arg(0));
+        // It is up to the user to process the environment variables
```

Comment:
Its not a stupid question, the problem is that I don't know what the data will be and I struggled to find a generic way to store and access it. I'm open to ideas and suggestions.
This is the recommended way to get the data https://bblanchon.github.io/ArduinoJson/faq/whats-the-best-way-to-use-the-library/ so we need to replace `struct SensorData` with some other storage